### DISCUSSION TITLE
ToggleSwitch with Icon storybook code

### DISCUSSION THREAD

**Discussion by NancyMcCallB** [iTwin/iTwinUI#1048] [Feedback]
Why do these show the exact same code?

![image](https://user-images.githubusercontent.com/45079789/216670584-8699f915-a59f-43bd-bfc6-05ee4fe42682.png)

**Comment by mayank99** [comment]
This is a bug in storybook that they are not very keen on fixing. It's one of our biggest frustrations with storybook, to the point that we are considering switching off of storybook. It's also why we are actively working on our new documentation website.

For now, if you want to get more useful examples out of storybook, you need to be on the "Canvas" tab and then double click the "Story" tab in the bottom panel.
![](https://user-images.githubusercontent.com/9084735/216672304-b4c6ce98-16e1-44fe-94a3-2929d69dee4f.png)
